extensions/amp-video/0.1/test/test-video-cache.js

```diff
@@ -139,14 +139,47 @@ describes.realWin('amp-video cached-sources', {amp: true}, (env) => {
       expect(addedSource.getAttribute('type')).to.equal('video/mp4');
     });
 
-    it('should add the sources sorted by bitrate', async () => {
```

Comment:
Can we keep this test (sorts by bitrate) + add another one (sorts by codec) + add another one (sorts by codec + bitrate)?
The issue with big tests that assert various things is that when they fail, it's not obvious which specific part actually broke.

Reply:
Also, let's add a test to check what happens when there's an unknown codec, not in our hardcoded list

Reply:
Updated. There are now 2 sorting tests: the original one that validates bitrate sorting logic, and a new one that validates codec sorting logic. The latter includes the sorting of a video with an "unknown" codec and asserts that this source is always sorted after the sources with VP9 and H.264 codecs.

Reply:
looks good!
I think we're still missing the last one `add another one (sorts by codec + bitrate)`, so far we only test sorting by bitrates or sorting by codecs, but not both

Reply:
Thanks, I've just added the third test!